TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -372,6 +372,15 @@ public override bool Visit(DataDefinition dataDefinition)
             return true;
         }
 
+        public override bool Visit(SourceComputer sourceComputer)
+        {
+            if (sourceComputer.Lines.Any(l => (l as CodeElementsLine)?.ScanState.WithDebuggingMode == true))
```

Comment:
Why use Lines property ?? `sourceComputer.CodeElement.DebuggingMode?.Value` should tell us whether debugging mode is active or not.

Reply:
This a scanner consequence of having WITH DEBUGGING MODE clause set within the SourceComputerParagraph code element of the SourceComputer node.

Yes a more explicit way would be to check that the consumed tokens of the SourceComputerParagraph code element contains at least the tokens DEBUGGING MODE.

Reply:
+1, checking property DebuggingMode of CodeElement SourceComputerParagraph is clearer.

Reply:
Sure

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/tree/94dd2d71d6dd982de8de0c257ff90aba9e55e698